.github/workflows/test.yml

```
@@ -0,0 +1,33 @@
+      - name: Install system dependencies
+        run: sudo apt-get update && sudo apt-get install -y libjpeg-dev zlib1g-dev
+
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install -r requirements.txt
+          pip install -r requirements-dev.txt
+
+      - name: Run tests
+        run: |
+          pytest
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)